vividus-extension-selenium/src/main/java/org/vividus/selenium/AbstractDesiredCapabilitiesConfigurer.java

```diff
@@ -19,18 +19,31 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Optional;
+import java.util.function.Consumer;
 
+import com.google.common.eventbus.Subscribe;
+
+import org.openqa.selenium.Proxy;
+import org.openqa.selenium.remote.CapabilityType;
 import org.openqa.selenium.remote.DesiredCapabilities;
 import org.vividus.bdd.context.IBddRunContext;
 import org.vividus.bdd.model.RunningStory;
+import org.vividus.selenium.event.WebDriverQuitEvent;
+import org.vividus.selenium.tunnel.TunnelManager;
+import org.vividus.selenium.tunnel.TunnelOptions;
 
-public abstract class AbstractDesiredCapabilitiesConfigurer implements DesiredCapabilitiesConfigurer
+public abstract class AbstractDesiredCapabilitiesConfigurer<T extends TunnelOptions>
```

Comment:
Why is Selenium configurer bind to tunnelling? What about usual Selenium Grid based on Docker? it doesn't support tunnelling